app/src/main/java/org/permanent/permanent/network/NetworkClient.kt

```diff
@@ -453,7 +453,7 @@ class NetworkClient(private var okHttpClient: OkHttpClient?, context: Context) {
     }
 
     fun updateMultipleRecords(records: MutableList<Record>): Call<ResponseVO> {
-        val request = toJson(RequestContainer().addRecords(records))
+        val request = toJson(RequestContainer().addRecords(records, isFolderRecordType = false))
```

Comment:
I would not hard code the boolean value, in order to be able to reuse this method.